halyard-deploy/src/test/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/GateBoot667ProfileFactoryTest.java

```diff
@@ -48,8 +64,32 @@ void setUp() {
   @Test
   void testGetGateConfigWithOAuth2Enabled() {
     GateProfileFactory.GateConfig config = factory.getGateConfig(mockServiceSettings, mockSecurity);
-    assertNotNull(config);
-    assertNotNull(config.getSpring());
-    assertNotNull(config.getSpring().getSecurity());
+    assertNotNull(
```

Comment:
move `oAuth2.setProvider(OAuth2.Provider.GOOGLE);` from the setUp method to here?

Reply:
done

https://github.com/spinnaker/halyard/pull/2216/commits/1bf30235912747c85e678d9a615c84a10d2fe08c

Reply:
done

https://github.com/spinnaker/halyard/pull/2216/commits/1bf30235912747c85e678d9a615c84a10d2fe08c